src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RequesterNameAndAddress.java

```diff
@@ -19,27 +19,31 @@
 
     private static final String KEY = "NAD";
     private static final String QUALIFIER = "PO";
-    private static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+    public static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
     private static final int REQUESTER_NAME_INDEX_IN_EDIFACT_STRING = 4;
 
     @NonNull
     private final String identifier;
     @NonNull
     private final HealthcareRegistrationIdentificationCode healthcareRegistrationIdentificationCode;
-    @NonNull
+
```

Comment:
If `requesterName` can be null, should we have the `@RequiredArgsConstructor` or `@AllArgsConstructor` decorator at the top?

Reply:
It's `final` that makes a parameter required. `@RequiredArgsConstructor` and `@AllArgsConstructor` will still do the same thing here.

Reply:
Ahh yeah no worries. Forgot it's required if it's `final` or `@NonNull`